x-pack/metricbeat/module/aws/elb/_meta/data.json

```diff
@@ -0,0 +1,33 @@
+{
+    "@timestamp": "2017-10-12T08:05:34.853Z",
+    "aws": {
+        "elb": {
```

Comment:
I wonder if we should place all cloudwatch metrics under `aws.cloudwatch.metrics` as we do with prometheus that we store all metrics under `prometheus.metrics`, so we can reuse dynamic mappings.
There are other fields to distinguish services, as `aws.cloudwatch.namespace` and `event.dataset`.